Core/Core/UIViews/CoreWebView.swift

```diff
@@ -189,32 +187,7 @@ open class CoreWebView: WKWebView {
               document.body.appendChild(script)
             }
 
-            // Replace all iframes with a button to launch in SFSafariViewController
-            document.querySelectorAll('iframe').forEach(iframe => {
-                const replace = iframe => {
-                    const a = document.createElement('a')
-                    a.textContent = \(CoreWebView.jsString(buttonText))
-                    a.classList.add('canvas-ios-lti-launch-button')
-                    a.href = iframe.src
-                    iframe.parentNode.replaceChild(a, iframe)
-                }
-                if (/\\/(courses|accounts)\\/[^\\/]+\\/external_tools\\/retrieve/.test(iframe.src)) {
-                    replace(iframe)
-                } else if (/\\/media_objects_iframe\\/m-\\w+/.test(iframe.src)) {
-                    const match = iframe.src.match(/\\/media_objects_iframe\\/(m-\\w+)/)
-                    if (match.length == 2) {
-                        const mediaID = match[1]
-                        const video = document.createElement('video')
-                        video.src = '/users/self/media_download?entryId='+mediaID+'&media_type=video&redirect=1'
-                        video.setAttribute('poster', '/media_objects/'+mediaID+'/thumbnail?width=550&height=448')
-                        video.setAttribute('controls', '')
-                        video.setAttribute('preload', 'none')
-                        iframe.parentNode.parentNode.replaceChild(video, iframe.parentNode)
-                    }
-                } else {
-                    iframe.addEventListener('error', event => replace(event.target))
```

Comment:
if this is happening multiple times, this could get triggered over & over, potentially making it take a long time to load.